webhook/webhook.py

```diff
@@ -127,7 +127,7 @@ async def webhook_send(
         await self.webhook_link_send(
             webhook_link,
             username=ctx.author.display_name,
-            avatar_url=ctx.author.avatar.url,
+            avatar_url=ctx.author.avatar.url if ctx.author.avatar.url else ctx.author.default_avatar.url,
```

Comment:
Please use [Member.display_avatar](https://discordpy.readthedocs.io/en/stable/api.html#discord.Member.display_avatar) rather than the if statement here.

Reply:
oh didn't realize, something like that existed.